widgetssdk/src/test/java/com/glia/widgets/core/fileupload/domain/RemoveFileAttachmentObserverUseCaseTest.java

```diff
@@ -0,0 +1,36 @@
+package com.glia.widgets.core.fileupload.domain;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.glia.widgets.core.fileupload.FileAttachmentRepository;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Observer;
+
+public class RemoveFileAttachmentObserverUseCaseTest {
+
+    private FileAttachmentRepository repository;
+    private RemoveFileAttachmentObserverUseCase subjectUnderTest;
+
+    @Before
+    public void setUp() {
+        repository = mock(FileAttachmentRepository.class);
+        subjectUnderTest = new RemoveFileAttachmentObserverUseCase(repository);
+    }
+
+    @Test
+    public void execute_callsRepositoryRemoveObserver_whenValidArgument() {
+        Observer observer = mock(Observer.class);
+        subjectUnderTest.execute(observer);
+        verify(repository).removeObserver(observer);
+    }
+
+    @Test
+    public void execute_callsRepositoryRemoveObserver_whenNullArgument() {
+        subjectUnderTest.execute(null);
+        verify(repository).removeObserver(null);
+    }
+}
```

Comment:
It seems to me that these tests doesn't bring much value. There is no logic in `RemoveFileAttachmentObserverUseCase` currently. There is nothing to test now. It may only be useful if there are plans for adding logic to `RemoveFileAttachmentObserverUseCase`. Do you agree? Am I missing anything?

Reply:
yep, will skip them from now on.